ParisBNF/abb/BNFabb29.xml

```diff
@@ -419,15 +413,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
   <msPart xml:id="p3">
         <msIdentifier/>
           <msContents>
-         <summary/>
```

Comment:
I am always in favour of empty `summary `elements, this will allow the app to print the summary